app/src/main/java/com/spencerpages/MainActivity.java

```diff
@@ -984,33 +959,145 @@ private void handleExportCollectionsPart2(){
         				break;
         			}
 
-        		}while(resultCursor.moveToNext());
+        		} while(resultCursor.moveToNext());
         	}
         	resultCursor.close();
-        	
-        	try {
-    			out.close();
-    		} catch (IOException e) {
-    			showCancelableAlert(mRes.getString(R.string.error_closing_output_file, file.getAbsolutePath()));
-    			return;
-    		}
-        	
+
+        	if(closeOutputFile(out, outputFile)){
+        	    return;
+        	}
+
         	if(errorOccurred){
         		return;
         	}
     	}
-    	
+
     	mDbAdapter.close();
-    	
+
 		showCancelableAlert(mRes.getString(R.string.success_export, EXPORT_FOLDER_NAME));
     }
-    
+
     private void showCancelableAlert(String text) {
 		mBuilder.setMessage(text).setCancelable(true);
 		AlertDialog alert = mBuilder.create();
 		alert.show();
 	}
 
+    private BufferedReader openFileForReading(File file){
+
+        FileInputStream fileInputStream;
+        InputStreamReader inputStreamReader;
+        BufferedReader bufferedReader;
+
+        try {
+            fileInputStream = new FileInputStream(file);
+            inputStreamReader = new InputStreamReader(fileInputStream, "UTF8");
+            bufferedReader = new BufferedReader(inputStreamReader);
+        } catch (Exception e) {
+            if(BuildConfig.DEBUG) {
+                Log.e(MainApplication.APP_NAME, e.toString());
```

Comment:
Let's have this log even in release mode - this way if we have a tech savvy user encounter this error (or, more likely, us) we can see what the issue is without having to install a debug build.  This case shouldn't be hit often, so it shouldn't put any strain on the log functionality